### PR TITLE
My Second Task on CSS

### PR DESCRIPTION
**Pull request by Chigoz169**
- Modified my index.html

- Added a CSS file to my index.html

### CODE REVIEW

File: style.css [hunk 1]
```diff
@@ -0,0 +1,533 @@
...
+  .container {
+    width: 100%;
+    margin: 0 auto;
+  }
+  .header {
+    border: 2px solid black;
+    display: flex;
+    flex-wrap: wrap;
+    width: 100%;
+    height: 198px;
```
Comment (Raphael2k6) [inline style.css]: You may consider using a relative unit

File: style.css [hunk 2]
```diff
@@ -0,0 +1,533 @@
...
+    display: block;
+    margin: 10px 10px;
+    font-size: 1.5625rem;
+    font-family: sans-serif;
+    color: #ffffff;
+  }
+  .footer__div__section__link:hover {
+    color: #0d237d;
+  }
+  .fa-lg {
```
Comment (Raphael2k6) [inline style.css]: What if a more descriptive class name is used.

File: style.css [hunk 3]
```diff
@@ -0,0 +1,533 @@
...
+    margin-top: 5px;
+  }
+  .footer__div__section__head {
+    font-size: 1.875rem;
+    padding: 0.3125rem;
+    color: #0d237d;
+  }
+  .footer__div__section__para {
+    font-size: 1.25rem;
+    width: 470px;
```
Comment (Raphael2k6) [inline style.css]: You may consider using a relative unit

**Review by Raphael2k6 — COMMENTED** [review]
You did a great job, your class names were descriptive and your styling elaborate. However, you may have to consider using more of relative units in your styling. Also, you may consider ensuring your CSS file is not too large as the larger your file, the longer it takes to render your page since CSS is a render-blocking file.
Similarly, you may need to bear in mind the differences in user devices and browser support for some features hence the need to include browser prefixes accordingly. Generally, you did a great job and I learnt some new things from your work.

cheers.

File: style.css [hunk 4]
```diff
@@ -0,0 +1,533 @@
...
+  font-family: "Franklin Gothic Medium", "Arial Narrow", Arial, sans-serif;
+}
+/* Media for Small Screens */
+@media screen and (max-width: 480px) {
+  .container {
+    width: 100%;
+    margin: 0 auto;
+  }
+  .header {
+    border: 2px solid black;
```
Comment (chiamakaikeanyi) [inline style.css]: Consider converting this unit to rem

File: style.css [hunk 5]
```diff
@@ -0,0 +1,533 @@
...
+    display: flex;
+    flex-wrap: wrap;
+    width: 100%;
+    height: 198px;
+    background-color: #ffffff;
```
Comment (chiamakaikeanyi) [inline style.css]: `#fff` can suffice

File: style.css [hunk 6]
```diff
@@ -0,0 +1,533 @@
...
+    text-decoration: none;
+    display: block;
+    margin: 5px 15px;
+    font-weight: 700;
+    font-size: 1.4375rem;
+    font-family: sans-serif;
+    color: #eb3e2f; 
+    }
+.footer__div__section__link:hover {
+    color: white;
```
Comment (chiamakaikeanyi) [inline style.css]: Preferably, use `#fff`

File: index.html [hunk 7]
```diff
@@ -4,127 +4,170 @@
...
-        <img src="images/ABJ.png" alt="A rectangular logo containing the letters ABJ." width="100px" height="40px"/>
-        <h1 id="index">ABRAHAM JUDAH</h1>
-        <nav>
-          <ul>
-            <li><a href="#index">Home</a></li>
-            <li><a href="#about">Profile</a></li>
-            <li><a href="#portfolio">Portfolio</a></li>
-            <li><a href="#contact">Contact</a></li>
+      <header class="header">
+        <img src="images/ABJ.png" alt="A rectangular logo containing the letters ABJ." width="100px"
```
Comment (CodyGbengz) [inline index.html]: Inline styling is considered bad practice. Please use a better approach.

File: index.html [hunk 8]
```diff
@@ -4,127 +4,170 @@
...
+          <a href="#" class="main__about__link">
+            Download Resume<i class="fas fa-arrow-right fa-sm"></i></a>
+        </div>
+        <div class="main__portfolio">
+          <h1 class="main__portfolio__head">My Projects</h1>
+          <section class="main__portfolio__projects">
+            <div></div>
+            <div></div>
+            <div></div>
+            <div></div>
```
Comment (CodyGbengz) [inline index.html]: These <div>s are empty. Can you remove them please.

File: index.html [hunk 9]
```diff
@@ -4,127 +4,170 @@
...
+            <a href="https://www.facebook.com" class="footer__div__section__link" target="_blank" rel="noopener">
+              <i class="fab fa-facebook-f fa-lg"></i>Facebook</a>
+            <a href="https://www.github.com" class="footer__div__section__link" target="_blank" rel="noopener">
+              <i class="fab fa-github fa-lg"></i>Github</a>
+            <a href="https://www.twitter.com" class="footer__div__section__link" target="_blank" rel="noopener">
+              <i class="fab fa-twitter fa-lg"></i>Twitter</a>
           </section>
         </div>
-        <p id="copy">&#169; Copyright 2020 ABJ. All Rights Reserved.</p>
+        <p class="footer__copy">&#169; Copyright 2020 ABJ. All Rights Reserved.</p>
```
Comment (CodyGbengz) [inline index.html]: I guess you'll have to update this section manually every year. What do you think will be a better approach?

File: style.css [hunk 10]
```diff
@@ -0,0 +1,533 @@
+/* Default Media */
+* {
+  box-sizing: border-box;
+}
```
Comment (CodyGbengz) [inline style.css:8]: Please add one line of space between code blocks for better readability.

File: index.html [hunk 11]
```diff
@@ -4,155 +4,171 @@
     <meta charset="UTF-8" />
     <meta name="viewport" content="width=device-width, initial-scale=1.0" />
     <meta name="keywords" content="Frontend, Web Developer" />
-    <meta
-      name="description"
-      content="A portfolio site created to showcase my personal projects which I have achieved so far in the technology industry."
+    <meta name="description" content="A portfolio site created to showcase my personal projects which I have achieved so far in the technology industry."
     />
     <title>Judah's Portfolio</title>
     <link rel="icon" type="image/png" href="images/A.png" />
```
Comment (Raphael2k6) [inline index.html:10]: you may consider naming your image properly

File: index.html [hunk 12]
```diff
@@ -4,155 +4,171 @@
...
-            <h2 class="main__description__title">Frontend Web Developer</h2>
+        </div>
+      </div>
+    </nav>
+    <!-- Introduction Section-->
+    <section class="showcase">
+      <div class="container">
+        <div class="container__div">
+          <div class="container__div__profile container__div__profile--pro1">
+            <img src="images/picture.jpg" alt="A personal profile picture." />
```
Comment (Raphael2k6) [inline index.html]: Your alt should describe the image

File: index.html [hunk 13]
```diff
@@ -4,155 +4,171 @@
...
+                  <li>Leadership Skill</li>
+                  <li>Teamwork</li>
+                  <li>Creative Thinking</li>
+                  <li>Problem Solving</li>
+                  <li>Flexible and Adaptable</li>
                 </ul>
-              </section>
+              </div>
+              <div class="container__block__flex__stack container__block__flex__stack--link">
+                <a href="www.googledocs.com" target="_blank" rel="noopener">Download Resume <i class="fas fa-arrow-right fa-sm"></i></a>
```
Comment (Raphael2k6) [inline index.html:127]: Does this really link to your resume

File: mobile.css [hunk 14]
```diff
@@ -0,0 +1,240 @@
...
+  display: inline-block;
+  border-radius: none;
+  border-right: none;
+  border-left: none;
+  padding: 0;
+  margin: 0;
+}
+
+.nav_bar a:hover {
+  background: #0d237d;
```
Comment (Raphael2k6) [inline mobile.css:72]: you may want to use a variable for repeated properties

**Review by Raphael2k6 — COMMENTED** [review]
generally, your naming is a very descriptive, great job. However, you may use variable to group repeated properties eg colors. Try to host your work on GitHub this will help me assess your styling and color theme better. thanks.

**Pull request merged**